.travis.yml

```diff
@@ -12,18 +12,21 @@ before_install:
   - dep ensure
 
 script:
-  - go vet
-  - golint -set_exit_status
-  - ginkgo -v --trace --race --cover --coverprofile coverprofile.out
-  - ginkgo -v --trace --race --cover --coverprofile coverprofile.out dark-node
+  - go vet ./...
+  - golint -set_exit_status `go list ./... | grep -Ev "(stackint/asm|vendor)"`
+  - ginkgo -v --trace --race --cover --coverprofile coverprofile.out stackint
   - ginkgo -v --trace --race --cover --coverprofile coverprofile.out identity
+  - ginkgo -v --trace --race --cover --coverprofile coverprofile.out shamir
+  - ginkgo -v --trace --race --cover --coverprofile coverprofile.out order
+  - ginkgo -v --trace --race --cover --coverprofile coverprofile.out compute
   - ginkgo -v --trace --race --cover --coverprofile coverprofile.out logger
   - ginkgo -v --trace --race --cover --coverprofile coverprofile.out network/dht
   - ginkgo -v --trace --race --cover --coverprofile coverprofile.out network/rpc
   - ginkgo -v --trace --race --cover --coverprofile coverprofile.out network
-  - ginkgo -v --trace --race --cover --coverprofile coverprofile.out orders
-  - ginkgo -v --trace --race --cover --coverprofile coverprofile.out shamir
+  - ginkgo -v --trace --race --cover --coverprofile coverprofile.out dark
+  - ginkgo -v --trace --race --cover --coverprofile coverprofile.out hyperdrive
+  - ginkgo -v --trace --cover --coverprofile coverprofile.out dark-node
```

Comment:
I removed race detection for the dark-node tests because the shutdown step has a data race when the channels are closed.

Reply:
Good idea, we will enable it again once the new processes/producers/consumers are in place.